### ISSUE TITLE
ImportWarning for tests/unit/utils

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#432]
Surfaces e.g. on Python 2.7 as follows:
```
/Users/maiera/PycharmProjects/pywbem/pywbemtools/tests/unit/cli_test_extensions.py:13: ImportWarning: Not importing directory '/Users/maiera/PycharmProjects/pywbem/pywbemtools/tests/unit/utils': missing __init__.py
    from .utils import execute_pywbemcli, assert_rc, assert_patterns, assert_lines
```

**Comment by andy-maier** [comment]
It turns out there is a utils.py module in that directory, and a utils subdirectory that is not used as a package, but by building the path to it in test_server_cmds.py. The ImportWarning is issued only with Python 2.7, but not with 3.7.

**Comment by andy-maier** [comment]
Addressed by PR #438.
